src/main/java/og_spipes/service/OntologyHelper.java

```diff
@@ -57,6 +59,22 @@ public Model createOntModel(File file) {
         return documentManager.getOntology(fileUri, ontModelSpec);
     }
 
+    public File findFileWhereStatementDefined(String s, PropertyImpl p, String o){
```

Comment:
Definitely not propertyImpl

Reply:
I do not see it done ... I see imports of both PropertyImpl and ResourceImpl ...